skunkworks_crow/src/main/res/values/colors.xml

```diff
@@ -5,4 +5,6 @@
     <color name="colorAccent">#00796B</color>
     <color name="colorTabInactive">#CFD8DC</color>
     <color name="colorTabActive">#FFFFFF</color>
+    <color name="colorButtonDisabled">#D3D3D3</color>
+    <color name="white">#FFFFFF</color>
```

Comment:
We discussed this @Chromicle in some other PR that avoid naming color resources with the actual color names, give some meaningful names to them, In this case we can remove the white color resource as you can directly use it like `android:color/white`

Reply:
yeah ! I updated